EnglishFilter/sections/foreign.txt

```diff
@@ -6452,27 +6452,33 @@ zoomit.ir#$#body > nav.is-sticky[style="top: 130px;"] { top: 0!important; }
 zoomit.ir#%#document.cookie = "BnShowNum=4; path=/;";
 ! soft98.ir - Persian Blocker rules (daily fix)
 !#if (!adguard_app_windows && !adguard_app_mac && !adguard_app_android && !adguard_ext_android_cb)
-soft98.ir#$?#:xpath(/html/body/div[*]/div/div[*]/div/div[1]):has(a[href*="insta"]) { remove: true; }
+soft98.ir#%#//scriptlet("remove-class", "c6e1y2", ".c6e1y2", stay)
 soft98.ir#$?#[href*="linkdoni.soft98.ir"] { remove: true; }
-soft98.ir#$?#:xpath(/html/body/div[*]/div[*]/aside/section[*]/header) { remove: true; }
-soft98.ir#$?#:xpath(/html/body/div[*]/div[*]/aside/div/section/header) { remove: true; }
+soft98.ir#$#[src*="file.soft98.ir"]{color: #437593 !important; background-color: #437593 !important; border: #437593 !important;}
+soft98.ir#$#[src*="cdn.hostdl.com"]{color: white !important; background-color: white !important; border: white !important;}
+soft98.ir,~forum.soft98.ir#$?#:xpath(/html/body/div[*]/div/div[*]/div/div[1]):has(a[href*="98"]){ remove: true; }
```

Comment:
Add space before `{ remove: true; }`. Btw, if you want to exclude `forum.soft98.ir`, try `www.soft98.ir` instead of 2 domains.
```
soft98.ir,~forum.soft98.ir#$?#:xpath(/html/body/div[*]/div/div[*]/div/div[1]):has(a[href*="98"]) { remove: true; }
```